aioxmpp/protocol.py

```diff
@@ -498,7 +498,7 @@ def _rx_feed(self, blob):
         except Exception:
             self._logger.exception(
                 "unexpected exception while parsing stanza"
-                " bubbled up through parser. stream so ded.")
+                " bubbled up through parser. stream so dead.")
```

Comment:
So this was actually deliberate as a kind of meme joke at the time. Probably good riddance though anyway :).